pages/common/wezterm.md

```diff
@@ -0,0 +1,37 @@
+# wezterm
+
+> Wez's Terminal Emulator, a powerful cross-platform terminal emulator and multiplexer written by wez and implemented in Rust.
+> Usage: wezterm [OPTIONS] [COMMAND].
+> More information: <https://wezterm.org>.
+
+- Start a new Wezterm process and create a window:
+
+`wezterm`
+
+- Establish an ssh session:
+
+`wezterm ssh <user@host:port>`
```

Comment:
```suggestion
`wezterm ssh {{user@host:port}}`
```
Angle brackets are reserved for links and keypresses. Use a [placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax)